app/assets/stylesheets/app/components/deprecated_card.scss

```diff
@@ -22,19 +22,19 @@
       opacity: 0.05;
     }
   }
+}
 
-  .deprecated-card-inner {
-    position: absolute;
-    display: block;
-    height: 100%;
-    width: 100%;
-    background-size: cover !important;
-    background-position: center !important;
-    opacity: 1;
-    @include transition-property(opacity);
-    @include transition-duration($transition-out-duration);
-    @include transition-timing-function($transition-function);
-  }
+.deprecated-card-inner {
+  position: absolute;
+  display: block;
+  height: 100%;
+  width: 100%;
+  background-size: cover !important;
+  background-position: center !important;
```

Comment:
As assets are phased out we can remove the terrible `deprecated-card` component. I don’t think it’s worth refactoring.